### PR TITLE
Update Readme with PyCon India Info

### PR DESCRIPTION
**Pull request by aktech**
@Sumith1896

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
 # PyConSymPy
```
Comment (sumith1896) [inline README.md]: hyphen would be great

**Comment by sumith1896** [comment]
+1, good to merge

File: README.md (hunk 1 above)
Comment (aktech) [inline README.md]: Thanks, Done!

**Comment by sumith1896** [comment]
Thanks @aktech

**Pull request merged**